GladiusEx.lua

```diff
@@ -14,13 +14,68 @@ local RC = LibStub("LibRangeCheck-3.0")
 local LSM = LibStub("LibSharedMedia-3.0")
 local fn = LibStub("LibFunctional-1.0")
 
+function UnpackAuraData2(auraData)
+  if not auraData then
+    return nil
+  end
+
+  local points = auraData.points
+  if (points ~= nil) then
+    points = unpack(auraData.points)

```

Comment:
I don't think this quite does what's needed, not that this matter much.
If it's not used we don't really need it
I think the `return` should have `unpack(auraData.points or {})` or the end?

Reply:
yeah, that's a lot better